framework-cloud/framework-openshift-operator/src/main/java/org/kie/cloud/openshift/operator/model/components/Status.java

```diff
@@ -27,6 +27,8 @@
     private String consoleHost;
     private Deployments deployments;
     private String phase;
+    private String version;
+    private Spec applied;
```

Comment:
just minor change request, can you move applied on the first position so the it's sorted. It's also better for checking of this Status class with status in KieApp

Reply:
Ok...